tests/test_user.py

```diff
@@ -83,7 +83,7 @@ def test_regenerate_verification_code(self, mocked_request):
         assert mocked_request.call_count == 1
         assert mocked_request.call_args == mock.call(
             content_type='application/json',
-            data='{"email": "test@test1234.com"}',
+            data='"test@test1234.com"',
```

Comment:
please update the test to pass in a dict - the test should serve as documentation